pulsebot.cfg.in

```diff
@@ -26,6 +26,7 @@ mozreview = webtools/reviewboard
 vcs = hgcustom/version-control-tools
 gfx = projects/graphics
 conduit = automation/conduit
+ci = build/ci-admin,build/ci-configuration
```

Comment:
#ci has mode n, which prevents posting messages while not joined, so this won't work without also adding #ci to channels above.